fuzzers/aflcc_no_orig_dic/fuzzer.py

```diff
@@ -0,0 +1,378 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for AFLcc fuzzer."""
+
+import shutil
+import os
+import threading
+import subprocess
+
+from fuzzers import utils
+
+from fuzzers.afl import fuzzer as afl_fuzzer
+
+
+def is_benchmark(name):
+    """ Check if the benchmark contains the string 'name' """
+
+    # TODO: return name in os.environ['BENCHMARK]
```

Comment:
Why all this hacky code, why not rely on BENCHMARK env variable as per this coment. Can you fix this in aflcc code as well.

Reply:
ahh, that's very old code when it was not stable. Changed.